src/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 0;
+  margin: 0 auto;
+  max-width: $calendar-width;
+  list-style: none;
+
+  @each $length, $days in $month-lengths {
+    &.calendar--length-#{$length} {
+      .calendar__day:nth-child(n + #{$days + 1}) {
+        display: none;
+      }
+    }
+  }
```

Comment:
The variable `$month-lengths` is used here but not defined within this file. Ensure it is defined elsewhere in your project or imported if defined in another file.